site/scripts/seed-db.ts

```diff
@@ -66,6 +66,11 @@ const script = async () => {
       preferredName: "Alice",
       email: "alice@example.com",
     },
+    {
+      shortname: "bob",
+      preferredName: "Bob",
+      email: "bob@example.com",
+    },
```

Comment:
Alice and Bob are users with no blocks. Logging in as Alice and viewing two profiles helps test empty page states.